src/containers/community/CommunityMembers.js

```diff
@@ -53,10 +56,11 @@ export default class CommunityMembers extends React.Component {
   }
 
   render () {
-    let { pending, people, location: { query }, community, currentUser, error } = this.props
+    let { pending, people, moderators, location: { query }, community, currentUser, error } = this.props
     if (error) return <AccessErrorMessage error={error}/>
     if (!currentUser) return <div>Loading...</div>
     let { search } = query
+    let subtitles = moderators.reduce((acc, mod) => ({...acc, [mod]: 'Moderator'}), {})
```

Comment:
i think it'll be more flexible in terms of styling if we just stick `isModerator` on each of the corresponding objects in `person` and then check for it in `PersonCard`. my first thought when i saw the screenshot was "that should be lighter gray (like .meta)" and the second thought was "we should show bios there"